docs/guides/push-work-pools.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Learn how to use Prefect push work pools to schedule work on serverless infrastructure without having to run a worker.
+tags:
+    - work pools
+    - deployments
+    - Cloud Run
+search:
+  boost: 2
+---
+
+# Push Work Pools <span class="badge cloud"></span>
+Push work pools are a special type of work pool that allows Prefect Cloud to submit flow runs for execution to serverless computing infrastruture without a user running a worker. Push work pools currently support execution in GCP Cloud Run Jobs, Azure Container Instances, and and AWS ECS Tasks.
+
+In this guide we'll:
+
+- Create a Push Work Pool that sends work to Google Cloud Run
+- Deploy a flow to that work pools
+- Execute our flow without having to run a worker or agent process to poll for flow runs
+
+## Google Cloud Run Setup
+
+To get the credentials we need to push work to Cloud run, we'll need a service account and an API Key.
+
+Create a service account by navigating to the service accounts page and clicking Create. Name and describe your aervice account, and click continue to configure permissions.
```

Comment:
Is it possible to link to this page?

Reply:
no but ill clarify its gcp service account and correct the typo